packages/components/src/upsell-menu-group/index.tsx

```diff
@@ -17,6 +17,7 @@ const Content = styled.div( {
 	width: 210,
 	whiteSpace: 'break-spaces',
 	maxWidth: '80vw',
+	fontSize: '0.8125rem',
```

Comment:
[Proposal]: Where did you get this size from? Based on some discussions in progress https://github.com/Automattic/wp-calypso/pull/81271, and in an effort to make things easier to track in the future, I suggest creating a new constant to indicate what this variable is. ( Searching for it, I noticed that this size is sometimes used for subtitles. )

Reply:
Thanks @kozer - I will take a look but likely I am going to remove it completely. I found another issue related to dependencies update and the nested popover that messed a bunch of CSS classes. I will attempt to fix the root cause instead because it affects more things that `fontSize`.